llama-index-integrations/readers/llama-index-readers-boarddocs/pyproject.toml

```diff
@@ -29,12 +29,12 @@ license = "MIT"
 maintainers = ["dweekly"]
 name = "llama-index-readers-boarddocs"
 readme = "README.md"
-version = "0.1.3"
+version = "0.1.4"
 
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
 llama-index-core = "^0.10.1"
-bs4 = "^0.0.2"
+beautifulsoup4 = ">=4.12.3,<5.0.0"
```

Comment:
are they the same pkg with different names?

Reply:
Yes they are the same package. beautifulsoup4 is the original while bs4 was created such that it would still work if people accidentally installed bs4 instead of beautifulsoup4